tests/toolkit/matl_buy_policy_tests.cc

```diff
@@ -181,12 +182,49 @@ TEST_F(MatlBuyPolicyTests, MultiReqQuantize) {
   ASSERT_FLOAT_EQ(req->target()->quantity(), quantize);
 }
 
-// Tests that agent cycles between active and dormant cycles during a mock
-// sim, only buying when active.
-TEST_F(MatlBuyPolicyTests, ActiveDormant) {
-  int active = 2;
-  int dormant = 3;
-  int dur = 4;
+TEST_F(MatlBuyPolicyTests, DefaultFixedActiveDormant) {
+  using cyclus::QueryResult;
+  
+  FixedIntDist a_dist = FixedIntDist(1);
+  FixedIntDist d_dist = FixedIntDist(-1);
+  FixedDoubleDist size_dist = FixedDoubleDist(1.0);
```

Comment:
Doesn't it make sense for this test to call `Init()` with no values for these and rely on the defaults defined in the header?  That will test both:

- that the defaults result in the default behavior
- the default behavior is to have no dormant phase